### PR TITLE
General partial code cleanup

### PR DESCRIPTION
**Pull request by porunov**
Mostly stylistic code cleanups produced by IntelijIDEA. 
Applied only some cleanups which I checked.

Signed-off-by: Oleksandr Porunov <alexandr.porunov@gmail.com>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### CODE REVIEW

File: janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/job/IndexRepairJob.java [hunk 1]
```diff
@@ -113,7 +113,7 @@ protected void validateIndexStatus() {
         }
         Preconditions.checkArgument(isValidIndex, "The index %s is in an invalid state and cannot be indexed. %s", indexName, invalidIndexHint);
         // TODO consider retrieving the current Job object and calling killJob() if !isValidIndex -- would be more efficient than throwing an exception on the first pair processed by each mapper
-        log.debug("Index {} is valid for re-indexing");
+        log.debug("Index {} is valid for re-indexing", index.name());
```
Comment (li-boxuan) [inline janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/job/IndexRepairJob.java:116]: index.name() will lead to a backend call.

Comment (porunov) [reply janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/job/IndexRepairJob.java:116]: Thanks for noticing it!
I can wrap into `if`. Something like:
```
if(log.isDebugEnabled()){
  log.debug("Index "+index.name()+" is valid for re-indexing");
}
```
Do you think that's OK or do you think we should simply remove / change the log message?

Comment (porunov) [reply janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/job/IndexRepairJob.java]: @li-boxuan I wrapped it into `if` block as commented above

File: janusgraph-core/src/main/java/org/janusgraph/util/system/BackgroundThread.java [hunk 2]
```diff
@@ -138,7 +138,7 @@ public void close(Duration duration) {
         try {
             join(maxWaitMs);
         } catch (InterruptedException e) {
-            log.error("Interrupted while waiting for thread {} to join",e);
+            log.error("Interrupted while waiting for thread {} to join", Thread.currentThread().getName(), e);
```
Comment (li-boxuan) [inline janusgraph-core/src/main/java/org/janusgraph/util/system/BackgroundThread.java:141]: What about just "getName()"?

**Review by li-boxuan — APPROVED** [review]
Thank you! I just left one nitpick, else looks good to me.

File: janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphOperationCountingTest.java [hunk 3]
```diff
@@ -549,8 +549,7 @@ public void testCacheConcurrency() throws InterruptedException {
         System.out.println("Hits: " + (getEdgeCacheRetrievals()-getEdgeCacheMisses()));
         System.out.println("Misses: " + getEdgeCacheMisses());
         assertEquals(numReads, lookups.get());
-        final int numLookupPropertyConstraints = numV;
-        assertEquals(2 * numReads + numV + numLookupPropertyConstraints, getEdgeCacheRetrievals());
+        assertEquals(2 * numReads + numV + numV, getEdgeCacheRetrievals());
```
Comment (rngcntr) [inline janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphOperationCountingTest.java:552]: This is a test so I think the additional variable is only there as an explanation of _why_ `numV` is added twice. Removing that would make me wonder why `numV` appears twice here.

File: janusgraph-backend-testutils/src/main/java/org/janusgraph/olap/OLAPTest.java [hunk 4]
```diff
@@ -424,7 +424,7 @@ public boolean requiresVertexPropertyAddition() {
     public static class DegreeCounter extends StaticVertexProgram<Integer> {
 
         public static final String DEGREE = "degree";
-        public static final MessageCombiner<Integer> ADDITION = (a,b) -> a+b;
+        public static final MessageCombiner<Integer> ADDITION = Integer::sum;
```
Comment (rngcntr) [inline janusgraph-backend-testutils/src/main/java/org/janusgraph/olap/OLAPTest.java:427]: I think we can also drop this and just use `Integer::sum` directly.

**Review by rngcntr — COMMENTED** [review]
LGTM!

**Review by rngcntr — APPROVED** [review]

**Comment by porunov** [comment]
Thank you @li-boxuan and @rngcntr ! Fixed those comments

File: janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java [hunk 5]
```diff
@@ -157,9 +157,9 @@ public static Mode parse(String mode) {
      * TODO Rename ZOOKEEPER_URL and "zookeeper-url" to ZOOKEEPER_URLS and
      * "zookeeper-urls" in future major releases.
      */
-    public static final ConfigOption<String[]> ZOOKEEPER_URL = new ConfigOption<String[]>(SOLR_NS,"zookeeper-url",
-            "URL of the Zookeeper instance coordinating the SolrCloud cluster",
-            ConfigOption.Type.MASKABLE, new String[] { "localhost:2181" });
+    public static final ConfigOption<String[]> ZOOKEEPER_URL = new ConfigOption<>(SOLR_NS, "zookeeper-url",
+        "URL of the Zookeeper instance coordinating the SolrCloud cluster",
```
Comment (li-boxuan) [inline janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java:161]: Can you revert this and the below indentation changes? It seems this indentation was by intention everywhere for CofigOptions. I would suggest fixing all or none.

Comment (porunov) [reply janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java]: Good catch! Fixed

**Pull request merged**